itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationOnHDFSEncryptedZones.java

```diff
@@ -102,12 +104,20 @@ public void targetAndSourceHaveDifferentEncryptionZoneKeys() throws Throwable {
           put(HiveConf.ConfVars.REPLDIR.varname, primary.repldDir);
         }}, "test_key123");
 
+    List<String> dumpWithClause = Arrays.asList(
```

Comment:
Why is this needed?

Reply:
Since the external table copy is happening on source, the configs are required to be passed at dump.